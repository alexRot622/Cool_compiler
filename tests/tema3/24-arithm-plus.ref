Loaded: trap.handler.nogc
The following symbols are undefined:
main

3
1
2
COOL program successfully executed
