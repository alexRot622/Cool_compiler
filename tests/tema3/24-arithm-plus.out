Loaded: trap.handler.nogc
The following symbols are undefined:
main

2
1
2
COOL program successfully executed
